### PR TITLE
Fix spell variants check in item summary renderer

### PR DESCRIPTION
**Pull request by In3luki**

### CODE REVIEW

File: src/module/actor/sheet/item-summary-renderer.ts [hunk 1]
```diff
@@ -195,7 +195,7 @@ export class CreatureSheetItemRenderer<AType extends CreaturePF2e> extends ItemS
                 }
 
                 if (actor instanceof CharacterPF2e) {
-                    if (chatData.variants) {
+                    if ((chatData.variants as Array<unknown>).length) {
```
Comment (stwlam) [inline src/module/actor/sheet/item-summary-renderer.ts:198]: Is that always going to be present for all items?

Comment (In3luki) [reply src/module/actor/sheet/item-summary-renderer.ts:198]: Currently yes. Alternatively I can set it to undefined if no variants exist and leave the check as it is.

Comment (stwlam) [reply src/module/actor/sheet/item-summary-renderer.ts:198]: That sounds better

Comment (In3luki) [reply src/module/actor/sheet/item-summary-renderer.ts]: Done

**Pull request merged**